checkov/terraform/evaluation/evaluation_methods/const_variable_evaluation.py

```diff
@@ -101,7 +101,9 @@ def _assign_definitions(self, assignment_files, folder):
         for var_file, variable_assignments in assignment_files.items():
             relative_var_file = f'/{os.path.relpath(var_file, self.root_folder)}'
             for definition_type in variable_assignments.keys():
-                for var_name, var_value in variable_assignments[definition_type]['assignments'].items():
+                for var_name, var_value in variable_assignments[definition_type].get('assignments', {}).items():
+                    if var_name == '':
+                        continue
```

Comment:
@schosterbarak Checkov was run agains a repository with a bad commit, which included the following statement in the variables block:
```
variable "" {
}
```
This variable does not have a name, and therefore cannot be referenced. It will now be skipped